### ISSUE TITLE
ServiceConfig for auth ?

### ISSUE THREAD

**Issue by kozak** [brendanhay/gogol#126]
Hi, Thanks for a great library!  
I've wanted to override the auth endpoint urls for testing/mocks. `Gogol` provides this ability for services via `ServiceConfig`. I was thinking of adding a similar feature to `Auth`. Would this be ok, or would you prefer a different way for overriding `tokenRequest` and `accountsRequest` (https://github.com/brendanhay/gogol/blob/develop/gogol/src/Network/Google/Internal/Auth.hs#L261-L287)? 
Cheers!

**Comment by piq9117** [comment]
Any chance you have a branch with this proposed change?

**Comment by tonicebrian** [comment]
Yes, we are in the process of releasing 1.0.0. I'll get to it ASAP
